modules/search/instant-search/lib/tracks.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { v4 as uuid } from 'uuid';
+
+const globalProperties = {};
+
+export function initializeTracks() {
+	window._tkq = window._tkq || [];
+}
+
+export function identifyUser( userId, userName ) {
+	window._tkq.push( [ 'identifyUser', userId, userName ] );
+}
+
+export function identifySite( siteId ) {
+	globalProperties.blog_id = siteId;
+}
+
+export function recordEvent( eventName, properties ) {
+	window._tkq.push( [ 'recordEvent', eventName, { ...globalProperties, ...properties } ] );
+}
+
+export function getRailcarIdPrefix() {
+	return `${ uuid().replace( /-/g, '' ) }`;
+}
+
+export function recordTrainTracksRender( properties ) {
+	recordEvent( 'jetpack_instant_search_traintracks_render', properties );
+}
+
+export function recordTrainTracksInteract( properties ) {
+	recordEvent( 'jetpack_instant_search_traintracks_interact', properties );
```

Comment:
This should have a property (`action`) that indicates it is a `click` event. This would let us also have event such as `add-to-cart` or `read-more` in the future.

Reply:
This has been added to `search-result-minimal.jsx`.